.github/workflows/check-new-library-versions.yml

```diff
@@ -0,0 +1,142 @@
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "Github Actions"
+          
+          issue_url=$(gh issue create --title "List unsupported library versions" --body "This issue lists unsupported versions of the existing libraries in the repo")
+          echo "::set-output name=issue::$issue_url"
+
+  test-all-metadata:
+    name: "🧪 ${{ matrix.coordinates }} (GraalVM for JDK ${{ matrix.version }} @ ${{ matrix.os }})"
+    permissions: write-all
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 20
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    needs: get-all-libraries
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.get-all-libraries.outputs.matrix) }}
+    steps:
+      - name: "☁️ Checkout repository"
+        uses: actions/checkout@v4
+      - name: "🔧 Setup java"
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'oracle'
+          java-version: '21'
+      - name: "🔧 Prepare environment"
+        uses: graalvm/setup-graalvm@v1
+        with:
+          set-java-home: 'false'
+          java-version: ${{ matrix.version }}
+          distribution: 'graalvm'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          native-image-job-reports: 'true'
+      - name: "Extract test path and library version"
+        run: |
+          LIBRARY_PATH=$(echo ${{ matrix.coordinates }} | cut -d ':' -f1-2 | sed 's/:/\//g')
+          LATEST_VERSION=$(find tests/src/$LIBRARY_PATH/* -maxdepth 1 -type d | sort -V | tail -1 | cut -d '/' -f5)
+          TEST_PATH="$LIBRARY_PATH/$LATEST_VERSION"
+          TEST_COORDINATES=$(echo "$TEST_PATH" | tr / :)
+          
+          echo "LATEST_VERSION=$LATEST_VERSION" >> ${GITHUB_ENV}
+          echo "TEST_PATH=$TEST_PATH" >> ${GITHUB_ENV}
+          echo "TEST_COORDINATES=$TEST_COORDINATES" >> ${GITHUB_ENV}
+      - name: "Pull allowed docker images"
+        run: |
+          ./gradlew pullAllowedDockerImages --coordinates=${{ env.TEST_COORDINATES }}
+      - name: "Disable docker"
+        run: |
+          sudo apt-get install openbsd-inetd
+          sudo bash -c "cat ./.github/workflows/discard-port.conf >> /etc/inetd.conf"
+          sudo systemctl start inetd
+          sudo mkdir /etc/systemd/system/docker.service.d
+          sudo bash -c "cat ./.github/workflows/dockerd.service > /etc/systemd/system/docker.service.d/http-proxy.conf"
+          sudo systemctl daemon-reload
+          sudo systemctl restart docker
+      - name: "🧪 Run '${{ env.TEST_COORDINATES }}' tests"
+        run: |
+          TESTING_VERSION=$(echo ${{ matrix.coordinates }} | cut -d ":" -f3)
+          export GVM_TCK_LV=$TESTING_VERSION
+
+          ./gradlew test -Pcoordinates=${{ env.TEST_COORDINATES }}
+      - name: "✔️ New library is supported"
+        if: success()
+        run: |
+          sh ./.github/workflows/tryPushVersionsUpdate.sh ${{ matrix.coordinates }} ${{ env.LATEST_VERSION }}
```

Comment:
Is there a reason for this being in a separate file? Also, the shebang of the file says it should run on `bash`, but you are invoking it with `sh`. Not sure this is a good idea.